drf_integrations/forms.py

```diff
@@ -33,26 +34,49 @@ def __init__(self, data=None, *args, **kwargs):
             except Registry.IntegrationUnavailableException:
                 pass
 
+        config = self.instance.get_config()
+
+        self.fields['api_client_name'].choices = [('-', '-')] + self.fields['api_client_name'].choices
```

Comment:
Why the "-" as a API name choice - and then later turning that dash into None?

Reply:
Added explanation in code. What happens is that django treats null and blank equally inside the form on render: if `api_client_name` is `None` the dropdown displays `Yoyo Wallet` (which in reality corresponds to `''`).